rest/src/main/java/com/redhat/ipaas/rest/v1/dao/ConnectorDAO.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.rest.v1.dao;
+
+import com.redhat.ipaas.rest.v1.controller.handler.rtdb.RealTimeDB;
+import com.redhat.ipaas.rest.v1.model.connection.Connector;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Named;
+
+@Named(Connector.KIND)
+@Service
+public class ConnectorDAO extends RealTimeAbstractDAO<Connector> {
```

Comment:
Your not going to actually have multiple impls of a DAO for an entity are you?  Right now per entity you pick a single DAO strategy.  For some we could store in the RTDB for other we could move the JPA for other we could use etcd :)